src/test/java/com/onfleet/demo/homework/manager/TaskAssignerTest.java

```diff
@@ -0,0 +1,91 @@
+package com.onfleet.demo.homework.manager;
+
+
+import com.onfleet.demo.homework.FixturedTest;
+import com.onfleet.demo.homework.TaskAssigner;
+import com.onfleet.demo.homework.collection.Tasklist;
+import com.onfleet.demo.homework.struct.Driver;
+import com.onfleet.demo.homework.struct.Task;
+import com.onfleet.demo.homework.util.ObjectGenerator;
+import org.junit.Assert;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+
+/**
+ * Supplies test methods for classes that comply with {@link com.onfleet.demo.homework.TaskAssigner}.
+ */
+abstract class TaskAssignerTest extends FixturedTest {
```

Comment:
abstract/general tests that test both `BlindTaskManager` and `TaskManager` in a consistent way